docs/source/gbd2017_models/causes/neoplasms/breast_cancer_with_stage0/cancer_model.rst

```diff
@@ -277,23 +277,66 @@ GBD does not give us any information on the prevalence or incidence of DCIS or L
       = Crude prevalence of ratio of LCIS X prevalence of breast cancer (prev_c429)
 
 
-.. todo::
+.. note::
 
-    - add age-specific ratios? Currently we do not have data for 65+ and for 2017 onwards.
-    - looking at revising the data from marketscan to see if we can get 65+ data. then will re-evaluate to see if we want to use age specific ratios. 
-    - A major assumption of this method of using ratios is that the ratio of DCIS or LCIS to breast cancer in the US population is the same as that in the Chinese population we are modelling. This could be a limitation if breast cancer manifests differently among racial groups. how to model 65+ ?? 
-    - How to obtain marketScan ratios for 2020-2040? 
-    - for those who are treated successfully do they stay in DCIS or remit back to susceptible? Need to read more literature (nicoly task)
-    - We might overestimate the total number of deaths due to breast cancer. According to GBD definition, patients are considered cured if they have survived more than 10 years after the mastectomy. However, the excess mortality rate still exists in simulation and generates extra deaths if we plan to run the model over 10 years.
+    - Currently we have applied non-age specific DCIS and LCIS prevalence ratios
+      from marketscan to inform DCIS and LCIS prevalence in 2020-2040.
+    - A major assumption of this method of using marketscan ratios is that the
+      ratio of DCIS or LCIS to breast cancer in the US population is the same as
+      that in the Chinese population we are modelling. This could be a limitation
+      if breast cancer manifests differently among racial groups.
+    - We believe that the crude prevalence ratio of DCIS and crude prevalence
+      ratio of LCIS are independent, meaning no correlation between these two
+      ratios.
+    - The draw-level DCIS and LCIS prevalence ratios can be generated based on
+      a truncated normal distirbution (lower bound = 0) with given mean and
```

Comment:
I think when the ratio is very close to the lower bound things will get weird, so let's keep an eye on this. If necessary, we can make the lower bound 0.1 to address weirdness.